lms/static/sass/features/_learner-profile.scss

```diff
@@ -164,53 +168,93 @@
 
     .wrapper-profile-sections {
         @extend .container;
-        padding: 0 ($baseline*1.5);
+        @include padding($baseline*1.5, $baseline*1.5, $baseline*1.5, 0);
+        min-width: 0;
+
+        @media (max-width: $learner-profile-container-flex) { // Switch to map-get($grid-breakpoints,md) for bootstrap
+            @include margin-left(0);
+            @include padding($baseline*1.5, 0, $baseline*1.5, 0);
+        }
+    }
+
+    .profile-header {
+        @include padding(0, 0, $baseline, $baseline*3);
+
+        .header {
+            @extend %t-title4;
+            @extend %t-ultrastrong;
+            display: inline-block;
+        }
+
+        .subheader {
+            @extend %t-title6;
+        }
     }
 
     .wrapper-profile-section-one {
-        width: 100%;
-        display: inline-block;
-        margin-top: ($baseline*1.5);
-        @include margin-left($baseline/2);
+        @include float(left);
+        @include margin-left($baseline*3);
+        width: 300px;
+        background-color: $white;
+        border-top: 5px solid $blue;
+        padding-bottom: $baseline;
+
+        @media (max-width: $learner-profile-container-flex) { // Switch to map-get($grid-breakpoints,md) for bootstrap
+            @include margin-left(0);
+            width: 100%;
+        }
     }
 
     .profile-section-one-fields {
-        @include float(left);
-        width: flex-grid(4, 12);
-        @include margin-left($baseline);
+        margin: 0 $baseline/2;
 
         .u-field {
-            margin-bottom: ($baseline/4);
-            padding-top: 3px;
-            padding-bottom: 3px;
-            @include padding-left(3px);
-        }
+            @extend %t-weight4;
+            @include padding(0, 0, 0, 3px);
+            color: $gray-dark;
 
-        .u-field-username {
+            .u-field-value {
+                @extend %t-weight4;
+                width: calc(100% - 40px);
 
-            input[type="text"] {
-                font-weight: 600;
+                .u-field-value-display {
+                    @extend %t-weight3;
+                }
             }
 
-            .u-field-value {
-                width: 350px;
-                @extend %t-title4;
+            .u-field-title {
+                color: $gray;
```

Comment:
I'm wondering about whether all the grays could be semantic variables too. Maybe $body-color?